test/e2e/accounts/snap-account-signatures.spec.ts

```diff
@@ -11,25 +11,30 @@ import {
 describe('Snap Account Signatures', function (this: Suite) {
   this.timeout(120000); // This test is very long, so we need an unusually high timeout
 
-  // Run sync, async approve, and async reject flows
-  // (in Jest we could do this with test.each, but that does not exist here)
   ['sync', 'approve', 'reject'].forEach((flowType) => {
-    // generate title of the test from flowType
     const title = `can sign with ${flowType} flow`;
 
-    it(title, async () => {
+    it(title, async function () {
+      this.retries(3); // Retry failed tests up to 3 times
+
       await withFixtures(
         accountSnapFixtures(title),
         async ({ driver }: { driver: Driver }) => {
           const isAsyncFlow = flowType !== 'sync';
 
+          console.log(`Starting test for ${flowType} flow`);
+
           await installSnapSimpleKeyring(driver, isAsyncFlow);
+          console.log(`Installed SnapSimpleKeyring for ${flowType} flow`);
```

Comment:
Sorry these logs are because I am trying to debug the failing e2e! I will remove them [98fc8b1](https://github.com/MetaMask/metamask-extension/pull/25191/commits/98fc8b12181c05fff9b9505e32259bfebe986cf3)